bloggies_frontend/src/NavBar/index.tsx

```diff
@@ -33,6 +34,39 @@ function NavBar() {
     history.push("/");
   };
 
+
+  const renderLoggedInNavLinks = () => {
+    if (user.membership_status === ACTIVE) {
+      return (
+        <>
+          <NavLink exact to={`/blogs/create`}>
+            compose blog
+          </NavLink>
+          <NavLink exact to={`/users/${user.id}/${urlDisplayName}`}>
+            my profile
+          </NavLink>
+          <NavLink exact to={`/users/${user.id}/settings`}>
+            premium settings
+          </NavLink>
+        </>
+      );
+    } else {
+      return (
+        <>
+          <NavLink exact to={`/blogs/create`}>
```

Comment:
Here, would we remove the create blogs feature from the free tier?

Reply:
Updated!